.github/workflows/ci.yml

```diff
@@ -1,18 +1,18 @@
-name: Java CI
-
-on: [push, pull_request]
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK 17
-        uses: actions/setup-java@v2
-        with:
-          java-version: '17'
-          distribution: 'adopt'
-          cache: maven
-      - name: Build with Maven
-        run: mvn --batch-mode --update-snapshots verify
+name: Java CI
+
+on: [push, pull_request]

```

Comment:
The solution looks good but it is strange that git shows us a lot of changes like that, with similar contents, Do you have any ideas what could have caused this?

Reply:
Sorry, i have no idea)